### DISCUSSION TITLE
Measuring methylation in one gene

### DISCUSSION THREAD

**Discussion by sr320** [RobertsLab/resources#1472] [Q&A]
What would be the best way to look at DNA methylation in a single gene (or small number of genes) in about ~40 fish (20 v 20 comparison)

**Comment by mgavery** [comment]
Probably an amplicon panel of bisulfite tx DNA to cover all CGs in a gene or set of genes. May not be able to get all CGs and lots of off target (do you have a genome?), but easier than using Nanopore for direct sequencing of target regions (they say you can do it using crispr, but easier said than done, I think).

**Reply by sr320** [reply]
Curious why you think direct sequencing with nanopore or pacbio would not be a viable option?

**Comment by mgavery** [comment]
Viable for genome-wide, but challenging if targeting specific region/gene.

**Reply by mgavery** [reply]
https://store.nanoporetech.com/us/cas9-sequencing-kit.html. ..genetics group was trying something like this, but not sure they got it to work.
